client/pages/acknowledgements.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import SharedLayout from "components/SharedLayout";
+import BackButton from "components/navigation/BackButton";
+import PlaceholderText from "components/mock/Placeholder";
+
+export default function Acknowledgements() {
+    return (
+        <SharedLayout>
+            <BackButton className="mt-3 mb-2" />
+            <h1>Acknowledgements</h1>
+            <PlaceholderText className="mt-5 mb-2" />
+        </SharedLayout>
+    )
+}
```

Comment:
I don't think there's an acknowledgements page in the wireframes, but we can keep this here and get rid of it later if we don't need it anymore.

Reply:
I was looking at outdated wireframes, please ignore me :)